planner/planner.go

```diff
@@ -6,21 +6,21 @@ package planner
 type Planner interface {
 	Name() string
 	IsRelevant(srcDir string) bool
-	Plan(srcDir string) *BuildPlan
+	GetPlan(srcDir string) *Plan
```

Comment:
I wanted to use `Plan` for the struct (since `BuildPlan` feels like a misnomer when we also use it to start the shell, and then `go` complained that the struct couldn't be called `Plan` at the same time as the method)

Thoughts?

Reply:
Ah, I see. I'd just go ahead and merge then. We can think about the names later.